attestation-service/attestation-service/Cargo.toml

```diff
@@ -13,36 +13,32 @@ snp-verifier = [ "verifier/snp-verifier" ]
 csv-verifier = [ "verifier/csv-verifier" ]
 cca-verifier = [ "verifier/cca-verifier" ]
 
-rvps-native = []
+# Only for testing and CI
```

Comment:
I think we'd better not add this comment, because maybe somebody wants to use it in some real deploy scene.

Reply:
Currently only CI and e2e testing integrate `built-in-rvps`. We might need to ensure from @surajssd if any "built-in-rvps" attestation service is used in their scenario/

Reply:
I don't think we use RVPS explicitly we just use whatever defaults exists.